README.md

```diff
@@ -68,11 +68,16 @@ There are various ways of contributing to this repository:
     an issue should contain links to suitable references, and ideally a precise
     informal statement of the conjecture.
 
-3.  **Improving the referencing and tagging of problems.** For example, adding
+3. **Formalise a problem already proposed**. You can find a list of problems ready
+   to be worked on in our [list of unassigned new conjectures](https://github.com/google-deepmind/formal-conjectures/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee+label%3A%22new+conjecture%22).
```

Comment:
Nice! As a drive-by, what do you think of changing the "and assign yourself!" to "ask to be assigned" or something along those lines in the "How to Contribute" guidelines below?